inspect_return_value.py

```diff
@@ -0,0 +1,172 @@
+"""
+<Program Name>
+  inspect_return_value.py
+
+<Author>
+  Sachit Malik <i.sachitmalik@gmail.com>
+
+<Started>
+  June 6, 2017
+
+<Copyright>
+  See LICENSE for licensing information.
+
+<Purpose>
+  Inspections constitute an important part of in-toto. In this script,
+  we inspect the return value of a step. The user supplies an integer
+  to the program(as compared to which the user wants to test various
+  boolean operators), along with path to the corresponding link file
+  (which link file he wants to test in), and also a bash style boolean
+  operator.
+
+  Suppose the link file is located at /user/abc/def/package.45gh325.link
+  and the user wants to check whether for the corresponding step (hence the
+  link file), the corresponding return field contains an integer less than 5.
+
+  The usage would be as follows:
+  python inspect_return_value.py -l /user/abc/def/package.45gh325.link -o lt 5
+
+  Please note: the various bash style boolean commands are listed as follows-
+  * bash-style-boolean-operator
+    --eq ... equal
+    --ne ... not equal
+    --lt ... less than
+    --le ... less than or equal
+    --gt ... greater than
+    --ge ... greater than or equal
```

Comment:
Please make sure that your documentation (docstrings, usage, ...) is consistent with how your tool actually works:
`--eq` != `--operator eq` == `--<operator>`